src/components/NextNPrevious.tsx

```
@@ -58,17 +19,15 @@
+                        <p>{`Next ${capitalize(postType).slice(0, -1)} ⇢`}</p>
                         {next.title}
                     </div>
                 </Link>
             )}
-        </NextPreviousItem>
+        </div>
     );
 };
 
-export default NextNPrevious;
+export default NextNPrevious;
```

ESLint

[error] Newline required at end of file but not found.

(style/eol-last)